stats/view/view_test.go

```diff
@@ -459,18 +459,6 @@ func TestViewRegister_negativeBucketBounds(t *testing.T) {
 	}
 }
 
-func TestViewRegister_zeroBucketBounds(t *testing.T) {
```

Comment:
Consider adding a test on verifying the leading 0 bucket bounds are dropped.

Reply:
added the test.